app/helpers/application_helper.rb

```diff
@@ -10,15 +10,11 @@ def homepage?
 
   # Map a method source file into a url to Github.com
   def github_url(ruby_doc)
-    version, file, line = ruby_doc.source_location.split(":")
+    _version, file, line = ruby_doc.source_location.split(":")
```

Comment:
I worked on this a little a few days ago and ended up writing the exact same thing. It sure makes me wish that `ruby_doc` had a reference to the version that the method is for. Then this could actually be an instance method on `ruby_doc`.

Like:

```ruby
ruby_doc.github_url
```

Reply:
I think this is a good direction to advocate for in the future!